web/src/components/tasks/business-structure/LegalStructureRadio.tsx

```diff
@@ -93,7 +93,9 @@ export const LegalStructureRadio = (props: Props): ReactElement => {
               value={state.profileData.legalStructureId || ""}
               onChange={handleLegalStructure}
             >
-              {LegalStructuresOrdered.map((legalStructure) => {
+              {/*           Changed map to not have foreign-nonprofit in legal structures loop.
+                Needed foreign-nonprofit added to legal structures for suffix tests to pass*/}
+              {LegalStructuresOrdered.filter((l) => l.id !== "foreign-nonprofit").map((legalStructure) => {
```

Comment:
As I commented above, I don't think `"foreign-nonprofit"` should be in the LegalStructures array at all.

Reply:
Taken out